trin-core/src/cli.rs

```diff
@@ -301,7 +344,7 @@ mod test {
         let expected_config = TrinConfig {
             web3_http_address: Url::parse("http://0.0.0.0:8080/").unwrap(),
             pool_size: 3,
-            web3_transport: "http".to_string(),
+            web3_transport: Web3TransportType::HTTP,
```

Comment:
You should be able to delete all of these `web3_transport` attribute definitions since it's already defined in `default()`. And while you're at it, might as well remove all of the `web3_http_address` definitions as well. In the cases where we're just re-defining the default value.

Reply:
Ahh, nvm. Just noticed that default is actually ipc. But you can still get rid some of the values in the tests below.

Reply:
I see what you mean but are you sure it's necessary ? Are we sure we will never change default values ?

I feel like we should write tests while abstracting default values. Also it helps make tests more readable as we see what we are testing.

Reply:
It's not necessary, but it helps remove unnecessary, redundant visual clutter from the tests. Imo, the default behavior is just that, the default behavior. It should be assumed in all of these tests. No doubt, default values will be changed, but the necessary adjustments to these tests can be made then.

Ultimately, I'd evaluate it on a case-by-case basis. If you think that redefining a default attribute is critical information to communicate to a reader trying to understand the test, then definitely leave it in. Otherwise, I'd cut it. Up to you how you want to handle these cases.